contracts/EllipseMarketMakerLib.sol

```diff
@@ -0,0 +1,310 @@
+pragma solidity 0.4.18;
+
+import './SafeMath.sol';
+import './IEllipseMarketMaker.sol';
+import './ERC20.sol';
+import './TokenOwnable.sol';
+
+/// @title Ellipse Market Maker Library.
+/// @dev market maker, using ellipse equation.
+/// @dev for more information read the appendix of the CLN white paper: https://cln.network/pdf/cln_whitepaper.pdf
+/// @author Tal Beja.
+contract EllipseMarketMakerLib is TokenOwnable, IEllipseMarketMaker {
+  using SafeMath for uint256;
+
+  // temp reserves
+  uint256 private l_R1;
```

Comment:
what's the prefix 'l' represents?

Reply:
local 
can be change to t for temp but it is not important